src/services/futures/trade.ts

```diff
@@ -0,0 +1,24 @@
+import { ethers } from "../..";
+import { FUTURES_TRACKING, PRICE_IMPACT_DELTA } from "./constants";
+import ISynthetixFuturesMarketV2 from "../../abi/ISynthetixFuturesMarketV2.json";
+
+export function getFuturesChangePositionTxData(
+  amount: ethers.BigNumber | string
+): string {
+  return new ethers.utils.Interface(
+    ISynthetixFuturesMarketV2.abi
+  ).encodeFunctionData("modifyPositionWithTracking", [
+    amount,
+    PRICE_IMPACT_DELTA,
+    ethers.utils.formatBytes32String(FUTURES_TRACKING)
+  ]);
+}
+
+export function getFuturesClosePositionTxData(): string {
+  return new ethers.utils.Interface(
+    ISynthetixFuturesMarketV2.abi
+  ).encodeFunctionData("closePositionWithTracking", [
```

Comment:
Can check in the Kwenta perps v2 UI, but I think they also use `submitOffchainDelayedOrderWithTracking` to close positions (smaller trading fee)

In other words, I think this is the only function that we need to support for any perps trading
(@dimlbc found this by testing the Kwenta Perps v2 UI)

Reply:
I changed it to submitOffchainDelayedOrderWithTracking and removed the close function.
The test cases won't go through on the fork, because I was not able to manipulate an order execution, but I tested it on live chain and it is working.

Reply:
Makes sense. Could skip the close test
<img width="354" alt="image" src="https://user-images.githubusercontent.com/44114383/218742252-8767ef60-3f94-40de-87e5-1d4e86409815.png">